modules/custom/openy_search/openy_google_search/src/Config/GoogleSearchOverrides.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\openy_google_search\Config;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactory;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+
+/**
+ * OpenY Google Search configuration override.
+ */
+class GoogleSearchOverrides implements ConfigFactoryOverrideInterface {
+
+  /**
+   * Configuration factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactory
+   */
+  private $configFactory;
+
+  /**
+   * Constructs a DomainConfigSubscriber object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactory $config_factory
+   *   Configuration factory.
+   */
+  public function __construct(ConfigFactory $config_factory) {
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names) {
+    $default_theme = $this->configFactory->getEditable('system.theme')->getOriginal('default');
```

Comment:
If we move the service to another module, then the logic here should be changed a bit, so that the base module verifies there is at least one of the submodules enabled.